client/src/config.rs

```diff
@@ -102,6 +102,10 @@ impl ClientBuilder {
         self
     }
 
+    /// Sets client to auto connect to the server session.
+    ///
+    /// `fail_fast` argument determines whether to early return with an error when encountering
+    /// issues during auto-connection to a relay on startup.
```

Comment:
We can add:
/// If `fail_fast` is set to true we do not reinitialize session with relay server if it gets lost.